bon-macros/src/builder/builder_gen/mod.rs

```diff
@@ -589,12 +589,18 @@ impl BuilderGenCtx {
 
     /// Name of the dummy struct that is generated just to give a name for
     /// the member in the error message when `IntoSet` trait is not implemented.
-    fn members_label(&self, member: &NamedMember) -> syn::Ident {
-        quote::format_ident!(
-            "{}__{}",
-            self.builder_ident.raw_name(),
-            member.setter_method_core_name()
-        )
+    fn members_label(member: &NamedMember) -> &syn::Ident {
```

Comment:
I'm not sure why this method was called plural "members" before. I guess it's something I missed during code review. But now I see that it should be just `member_label`, or maybe this method can be removed completely now that it just delegates to a `setter_method_core_name()`.

Same thing about plural naming applies to `absolute_members_label` (it should be `absolute_member_label`).